agent-skills/atmos-workflows/SKILL.md

```
@@ -0,0 +1,449 @@
+  # Can also be set using ATMOS_WORKFLOWS_BASE_PATH env var or --workflows-dir flag
+  base_path: "stacks/workflows"
+```
+
+If `base_path` is set at the root level of `atmos.yaml`, the `workflows.base_path` is resolved
+relative to it.
+
+### Directory Structure
+
+Organize workflow files by purpose, environment, or team:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)